pom.xml

```diff
@@ -1170,5 +1193,19 @@ under the License.</licenseText>
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>pmd</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
this is not working, why? only giving profile `cis -Ppmd` activates PMD. So this switch seems not to work.
@gnodet